src/components/approval-tasks/ApprovalTaskActionDropdown.tsx

```diff
@@ -47,7 +50,8 @@ const ApprovalTaskActionDropdown: React.FC<ApprovalTaskActionDropdownProps> = ({
     launchModal(approvalModal, {
       resource: approvalTask,
       pipelineRunName: pipelineRun?.metadata?.name,
-      userName: currentUser,
+      userName: currentUser.username,
```

Comment:
null checks are needed

```suggestion
      userName: currentUser?.username,
```